src/UI/NavigationContainer.elm

```diff
@@ -515,6 +515,26 @@ toBrowserDocument cfg page (Navigator model) =
     }
 
 
+{-| End of the builder's life.
+The result of this function is a ready-to-insert Elm UI's Element.
+
+**NOTE**: This is a way to work around the NavigationContainer to display only the content of the container. It's meant to be used in the project's showcase and only works for desktop.
+
+    someView
+        |> Nav.contentSingle
+        |> Nav.toShowcaseElement
+
+-}
+toShowcaseElement : Content msg -> Element msg
```

Comment:
I don't like this, not even a bit.
What if... we move "Content" to an Internal type, with this... and then use this internal in the showcase?